pyproject.toml

```diff
@@ -54,10 +54,10 @@ path = "dbt/adapters/postgres/__version__.py"
 
 [tool.hatch.envs.default]
 dependencies = [
-    "dbt-adapters @ git+https://github.com/dbt-labs/dbt-adapters.git",
+    "dbt-adapters @ git+https://github.com/dbt-labs/dbt-adapters.git@base-microbatch-tests",
```

Comment:
I assume at this point we're just waiting for https://github.com/dbt-labs/dbt-adapters/pull/300 to then get this line updated. Is that correct?

Reply:
yep!